group/ristretto255.go

```diff
@@ -0,0 +1,179 @@
+package group
+
+import (
+	"crypto"
+	"io"
+
+	h2c "github.com/armfazh/h2c-go-ref"
+
+	r225 "github.com/bwesterb/go-ristretto"
+)
+
+var (
+	Ristretto255 Group = ristrettoGroup{}
+)
+
+type ristrettoGroup struct {
+}
+
+type ristrettoElement struct {
+	p *r225.Point
+}
+
+type ristrettoScalar struct {
+	s *r225.Scalar
+}
+
+func (g ristrettoGroup) NewElement() Element {
```

Comment:
Is there the expectation that the new element should be zero?  Because as it's initialised now, it's in an undefined state.

Reply:
I'm not sure -- @armfazh?

Reply:
Yes, it is expected New calls the Identity method.
e.g.,
```func (g wG) NewElement() Element { return g.Identity() }```